Parametrize remaining tests in TestAutogradFunctional to use logging_tensor

soulitzer opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #73970

Differential Revision: [D34851561](https://our.internmc.facebook.com/intern/diff/D34851561)

soulitzer committed:
Parametrize remaining tests in TestAutogradFunctional to use logging_tensor

[ghstack-poisoned]

soulitzer committed:
Update on "Parametrize remaining tests in TestAutogradFunctional to use logging_tensor"




[ghstack-poisoned]

albanD reviewed (approved):
Nice!

looking forward to the investigations on all of these failures!

soulitzer commented:
@soulitzer has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34851561).

soulitzer committed:
Update on "Parametrize remaining tests in TestAutogradFunctional to use logging_tensor"


Differential Revision: [D34851561](https://our.internmc.facebook.com/intern/diff/D34851561)

[ghstack-poisoned]

soulitzer committed:
Update on "Parametrize remaining tests in TestAutogradFunctional to use logging_tensor"


Differential Revision: [D34851561](https://our.internmc.facebook.com/intern/diff/D34851561)

[ghstack-poisoned]

soulitzer commented:
@pytorchbot merge this
